Arrangement_on_surface_2/include/CGAL/Arr_geometry_traits/Bezier_cache.h

```diff
@@ -606,21 +598,21 @@ bool _Bezier_cache<NtTraits>::_intersection_params
 
   for (k = degX_2; k >= 0; k--)
   {
-    coeff = nt_traits.get_coefficient (polyX_2, k) * normX_1;
+    coeff = nt_traits.get_coefficient (polyX_2, k);
```

Comment:
Multiplying the polynomials with these huge normalizing factors really degraded the root finding algorithm. I can't see any reason why it is done. Shouldn't it be divided by it to get the original rational polynomial?

Removing these seemed to solve the problem. If they're necessary, I have another solution that works.